src/Commands/Upstream/Updates/ApplyCommand.php

```diff
@@ -42,18 +42,27 @@ public function applyUpstreamUpdates($site_env, $options = ['updatedb' => false,
         }
 
         $updates = $this->getUpstreamUpdatesLog($env);
+        $composerUpdates = $this->getComposerUpdatesLog($env);
+
         $count = count($updates);
-        if ($count) {
+        $composerCount = count($composerUpdates);
+        if ($count || $composerCount) {
+            $prefix = sprintf("Applying %d upstream update(s)", $count);
+            if ($composerCount) {
+                $prefix .= sprintf(" and %d composer updates", $composerCount);
```

Comment:
Maybe change `" and %d composer updates"` to `" and %d composer update(s)"`, similar to `"upstream update(s)"`?